.changelog/11446.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
Thanks for adding the changelog!

Something to address later (once you get a review back from engineering): this changelog file should match the PR number (`.changelog/11461`). This currently overwrites a different (existing) changelog entry.

Reply:
Ops! I wanted to be compliant with the style/process and I think I overdid it! 
OK, I am gonna remove it for now then and I'll wait for further instructions. 
Thanks @jkirschner-hashicorp!

Reply:
You did the right thing! You just need to change the filename :) Should be `.changelog/11461.txt`, not `.changelog/11446.txt`.

Reply:
I currently have a PR open to update the contributing docs to make the changelog entry creation process clearer: https://github.com/hashicorp/consul/pull/11391

I hope to get it merged soon!

Reply:
Mhh, not sure if `config` is the right code area 🤔, it's not a new config option even if the change affects config options. Thoughts?

Reply:
I think type `feature` and code area `config` would be used for a new config option. Type `improvement` area `config` feels appropriate here to me!

Reply:
Awesome, thanks @jkirschner-hashicorp